src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderThirdPartyPackage.java

```diff
@@ -6,9 +6,9 @@
 import java.*;
 import javax.swing.*;
 
-import static sun.tools.util.CommandLine.parse;
-import static sun.tools.util.ModifierFilter.ALL_ACCESS;
-import static sun.tools.util.ModifierFilter.ALL_ACCESS;
+import static com.puppycrawl.tools.checkstyle.internal.utils.CheckUtil.getTokenText;
+import static com.puppycrawl.tools.checkstyle.internal.utils.CheckUtil.getTokenText;
+import static com.puppycrawl.tools.checkstyle.internal.utils.TestUtil.parseFile;
```

Comment:
Inputs should not rely on our production code. See #4845

Reply:
Thank you! I was looking for something like that, but did not succeeded.
If there is a dedicated issue for import tests, then this PR should be split.

Reply:
Done.